tokens/blau.json

```diff
@@ -1049,44 +1059,44 @@
       "description": "blauPurple30"
     },
     "textLinkDanger": {
-      "value": "{palette.blauRed}",
+      "value": "{palette.blauRed40}",
       "type": "color",
-      "description": "blauRed"
+      "description": "blauRed40"
     },
     "textLinkSnackbar": {
       "value": "{palette.blauPurple30}",
       "type": "color",
       "description": "blauPurple30"
     },
     "textActivated": {
-      "value": "{palette.blauBlueSecondary}",
+      "value": "{palette.blauBluePrimary}",
       "type": "color",
-      "description": "blauBlueSecondary"
+      "description": "blauBluePrimary"
     },
     "textBrand": {
       "value": "{palette.blauBluePrimary}",
       "type": "color",
       "description": "blauBluePrimary"
     },
     "inputBorder": {
-      "value": "{palette.grey5}",
+      "value": "{palette.darkModeGrey5}",
       "type": "color",
-      "description": "grey5"
+      "description": "darkModeGrey5"
     },
     "control": {
-      "value": "{palette.darkModeGrey6}",
+      "value": "{palette.grey4}",
```

Comment:
`control` and `controlInverse` have different values in darkMode, both should have the same

Reply:
fixed